LayoutTests/fast/css/cssText-shorthand.html

```diff
@@ -31,27 +26,25 @@
     ['list-style-type: circle; list-style-position: inside; list-style-image: initial;', 'list-style: inside circle;'],
     ['margin-top: 1px; margin-right: 2px; margin-bottom: 3px; margin-left: 4px;', 'margin: 1px 2px 3px 4px;'],
     ['outline-width: 2px; outline-style: dotted; outline-color: blue;', 'outline: blue dotted 2px;'],
-    ['overflow-x: scroll; overflow-y: hidden;', 'overflow: scroll hidden;'],
+    ['overflow-x: scroll; overflow-y: hidden;', 'overflow-x: scroll; overflow-y: hidden;'],
```

Comment:
That's pretty strange, overflow uses `get2Values`, so how can a change in `get4Values` affect it? Also , `overflow: scroll hidden` seemed correct.